test/tables/ltag.js

```diff
@@ -1,6 +1,6 @@
 import assert from 'assert';
-import { hex, unhex } from '../testutil';
-import ltag from '../../src/tables/ltag';
+import { hex, unhex } from '../testutil.js';
+import ltag from '../../src/tables/ltag.js';
```

Comment:
Why are we adding js to the end of everything?

Reply:
to make it work on all platform natively (web browser import don't automatically append .js)